docs/advanced-guides/cloud-in-a-box.md

```diff
@@ -73,15 +74,71 @@ system release changes.  The installation of historical releases is currently no
    * Remove the USB storage device
      (The USB stick is only needed again if the Cloud in a Box system is to be fully reinstalled.)
    * Connect the first network interface to an ethernet interface that provides access to the internet via DHCP configuration
-   * Boot the system from the internal hard disk device (if 
+   * Boot the system from the internal hard disk device
 6. The deployment will start. This takes some time and the system will shutdown when the
    deployment is finished. This takes roughly an hour, possibly longer depending on the
    hardware and internet connection.
 
-7. Start the system again. System is ready for use, by default DHCP is tried on the first network device.
+### Manual installation
+
+:::note
+Using the autoinstall above is recommended.
+:::
 
-8. Login via SSH. Use the user `dragon` with the password `password`.
-   (You can obtain the ip address by inspecting the logs of your dhcp server or from the *issue text* of the virtual consoles of the system)
+1. Follow the [manual provisioning guide](../guides/deploy-guide/bootstrap.md#manual-provisioning), skip the part about
```

Comment:
Its good that you reference available documentation, but unfortunately this breaks when documentation is deployed https://docs.scs.community/docs/iaas/deployment-examples/cloud-in-a-box/advanced-guides/cloud-in-a-box/. I think its the best that we use absolute references.